commit 94c9e5b9fc70f3e30fa04cd6b3d45b892e8d91c4 (refs/changes/17/817/1)
Author: Todd Whiteman <todd.whiteman@joyent.com>
Date:   2016-10-28T19:11:31-07:00 (2 years, 11 months ago)
    
    DOCKER-958 Change 'make test-integration-in-coal' to only run against latest docker

diff --git a/test/runtest.common b/test/runtest.common
index 70aa5bf..cca242e 100644
--- a/test/runtest.common
+++ b/test/runtest.common
@@ -54,9 +54,13 @@ export NAPI_URL="http://$(vmadm lookup -j alias=napi0 | json 0.nics \
 #    run the "test/integration/cli-*.test.js" files. This is overridable, e.g.:
 #       DOCKER_CLI_VERSIONS=1.6.2 .../runtests
 export DOCKER_AVAILABLE_CLI_VERSIONS="1.12.2 1.11.1 1.10.3 1.9.1 1.8.3 1.7.1 1.6.2"
-if [[ -z "$DOCKER_CLI_VERSIONS" ]]; then
+if [[ "$DOCKER_CLI_VERSIONS" == "all" ]]; then
     export DOCKER_CLI_VERSIONS="$DOCKER_AVAILABLE_CLI_VERSIONS"
 fi
+if [[ -z "$DOCKER_CLI_VERSIONS" || "$DOCKER_CLI_VERSIONS" == "latest" ]]; then
+    # Use the first (latest) version, splitting on the space separator.
+    export DOCKER_CLI_VERSIONS="${DOCKER_AVAILABLE_CLI_VERSIONS%% *}"
+fi
 
 # Guard
 guard_file=/lib/sdc/.sdc-test-no-production-data
